RetsTalk/RetsTalk/Retrospect/Model/RetrospectManager.swift

```diff
@@ -9,41 +9,124 @@ import Foundation
+        let isProgressEntities = try await retrospectStorage.fetch(by: isProgressRequest)
+        
+        let finishedRecentRequest = finishedRecentRetrospectFetchRequest(offset: offset, amount: amount)
+        let finishedRecentEntities = try await retrospectStorage.fetch(by: finishedRecentRequest)
         
+        let resultRetrospects = [isPinnedEntities, isProgressEntities, finishedRecentEntities]
+            .flatMap { $0 }
+        retrospects.append(contentsOf: resultRetrospects)
     }
     
     func create() -> RetrospectChatManageable {
         let retropsect = Retrospect(userID: userID)
         let retrospectChatManager = RetrospectChatManager(
             retrospect: retropsect,
-            persistent: CoreDataManager(name: "RetsTalk", completion: { _ in }),
-            assistantMessageProvider: CLOVAStudioManager(urlSession: .shared),
+            persistent: retrospectStorage,
+            assistantMessageProvider: assistantMessageProvider,
             retrospectChatManagerListener: self
         )
         retrospects.append(retropsect)
         
         return retrospectChatManager
     }
     
-    func update(_ retrospect: Retrospect) {
+    func update(_ retrospect: Retrospect) async throws {
+        
+    }
+    
+    func delete(_ retrospect: Retrospect) async throws {
+        
+    }
+}
+
+// MARK: - ChatManager Create FetchRequest
+
+extension RetrospectManager {
+    private func isPinnedRetrospectFetchRequest() -> PersistfetchRequest<Retrospect> {
+        let predicate = NSCompoundPredicate(andPredicateWithSubpredicates: [
+            NSPredicate(format: "userID = %@", argumentArray: [userID]),
+            NSPredicate(format: "isPinned = %@", argumentArray: [true]),
+        ])
+        let sortDescriptors = NSSortDescriptor(key: "createdAt", ascending: false)
+        
+        let request = PersistfetchRequest<Retrospect>(
+            predicate: predicate,
+            sortDescriptors: [sortDescriptors],
+            fetchLimit: Metrics.isPinnedFetchAmount
+        )
+        
+        return request
+    }
+    
+    private func isProgressRetrospectFetchRequest() -> PersistfetchRequest<Retrospect> {
+        let predicate = NSCompoundPredicate(andPredicateWithSubpredicates: [
+            NSPredicate(format: "userID = %@", argumentArray: [userID]),
+            NSPredicate(format: "status != %@", argumentArray: [Texts.finishedStatus]),
+        ])
+        let sortDescriptors = NSSortDescriptor(key: "createdAt", ascending: false)
+        
+        let request = PersistfetchRequest<Retrospect>(
+            predicate: predicate,
+            sortDescriptors: [sortDescriptors],
+            fetchLimit: Metrics.isProgressFetchAmount
+        )
         
+        return request
     }
     
-    func delete(_ retrospect: Retrospect) {
+    private func finishedRecentRetrospectFetchRequest(offset: Int, amount: Int) -> PersistfetchRequest<Retrospect> {
```

Comment:
```suggestion
    private func recentFinishedRetrospectFetchRequest(offset: Int, amount: Int) -> PersistfetchRequest<Retrospect> {
```
finished 앞에 recent가 붙는게 더 자연스러운 느낌이 드네요!